.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: python
 
+sudo: required
+
+services:
+  - docker
+
 cache:
   directories:
   - ~/.cache/pip
 
 python:
-- 2.7
 - 3.7
 
 install:
 - pip install -e .
```

Comment:
The line `pip install -e .` runs inside the command `docker-compose build`, precisely in the step 5.
So it is assured the invocation of it.
Reference: (https://github.com/GlobalFishingWatch/pipe-tools/blob/66-travis-drop-py27/Dockerfile#L7)
@pwoods25443 Could it be possible to remove this line?

Reply:
ok - I'll re-base to `develop` and try removing the `pip install -e` from the travis build